Add Unauthorized Access Page with icon and navigation

Mohammedamaan5714 opened the pull request:
This pull request addresses Issue #31: Frontend: Design and Implement Unauthorized Access Page.

- A clear 403 Unauthorized message
- A lock icon using react-icons
- A message indicating restricted access
- Basic navigation support if needed

Changes Made

- Created a new component: UnauthorizedPage.jsx
- Added icon: FaLock from the react-icons/fa package
- Minor styling using Tailwind CSS (if applicable)

Additional Notes-

To support the use of icons, please run:

npm install react-icons

Let me know if any adjustments are needed. Looking forward to your review!

Ayush0316 commented:
**@Mohammedamaan5714**
As I already mentioned in our project's Discord server, if you need to use any button, you **must use the `Button` component** that is already implemented in our project—**do not use the native HTML `<button>` tag**.
Also, please don't push the `.idea` folder, as it is not related to the project.

Mohammedamaan5714 commented:
###  Unauthorized Access Page

Closes #31

- Added a clean and responsive Unauthorized (403) page
- Used the shared `Button` component as instructed
- Included a short friendly message and navigation option
- Added a relevant icon for better UX
- `.idea/` folder has been added to `.gitignore` to avoid unnecessary project files

Please review and let me know if anything needs to be changed 🙌

Mohammedamaan5714 commented:
###  Unauthorized Access Page

Closes #31

- Added a clean and responsive Unauthorized (403) page
- Used the shared `Button` component as instructed
- Included a short friendly message and navigation option
- Added a relevant icon for better UX
- `.idea/` folder has been added to `.gitignore` to avoid unnecessary project files

Please review and let me know if anything needs to be changed 🙌

Ayush0316 commented on AppUi/src/pages/unauthorized.jsx:
```diff
@@ -1,10 +1,23 @@
+      <p className="text-gray-600 mb-6">You don’t have permission to view this page.</p>
+
+      <div className="flex gap-4">
+        <Button onClick={() => navigate("/")}>Go to Home</Button>
```
As it is already mentioned in contributing.md file, don't use magic strings. 
Use `GUEST_PATH.LOGIN` instead

Mohammedamaan5714 replied:
okay sir
i changes /login to GUEST_PATH 
and the ui is look like this 
<img width="1919" height="845" alt="image" src="https://github.com/user-attachments/assets/5b248f30-4892-4e14-8f9e-7ca500cc348f" />

if you think its fine then ill send another pr request
and if you need any changes let me know

Ayush0316 replied:
Our web app uses a glassy theme, so please make sure to follow that design style.
Use glassmorphism effects where appropriate.

For reference, check the glassy look of the container in the screenshot below. That’s the kind of effect we’re aiming for:
<img width="1901" height="909" alt="image" src="https://github.com/user-attachments/assets/200629aa-dacd-4e5c-a367-5c261e380a81" />

Ayush0316 commented on AppUi/src/pages/unauthorized.jsx:
```diff
@@ -1,10 +1,23 @@
+
+      <div className="flex gap-4">
+        <Button onClick={() => navigate("/")}>Go to Home</Button>
+        <Button onClick={() => navigate("/login")} variant="secondary">Login</Button>
```
Use `GUEST_PATH.LANDING`.

Ayush0316 reviewed (commented):
Also for any UI related changes always add screenshots or demo video of our changes
